openshift/templates/eagle-api.dc.json

```diff
@@ -441,7 +478,7 @@
                   },
                   "limits": {
                     "cpu": "${NODEJS_CPU_LIMIT}",
-                    "memory": "${NODEJS_MEMORY_LIMIT}"
+                    "memory": "750m"
```

Comment:
Don't we have this var now?

Reply:
That's a special case for the generator pod.  It needed more RAM than the base API pod to do the data generator.  The pod isn't used afterwards so I didn't want to add another param and env to the template

Reply:
aah, okay sounds good to me 👍